server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -1441,4 +1441,39 @@ public void setMailboxesShouldUpdateMailboxNameWhenRenamingAChildMailbox() {
             .body(ARGUMENTS + ".list", hasSize(1))
             .body(ARGUMENTS + ".list[0].name", equalTo("mySecondBox"));
     }
+
```

Comment:
What happens if I rename a mailbox "any" to a system mailbox ? MailboxAlreadyExist exception ?

Maybe a test about this might be nice, as it is some kind of corner case ....